frontend/src/lib/utils/candidateStore.ts

```diff
@@ -14,14 +20,18 @@ export interface CandidateContext {
   loadLocalStorage: () => void;
   emailOfNewUserStore: Writable<string | null | undefined>;
```

Comment:
It'd be nice to do some renaming here and elsewhere by removing 'Store' from store name because it's not common practice elsewhere and the IDE will tell us whether something is a store or not. Thus:

`emailOfNewUserStore` => `emailOfNewUser` or even `newUserEmail`.